css/main.css

```diff
@@ -156,36 +192,120 @@ body {
 
 /*Footer*/
 
+.footer-title {
+    text-align: center;
+}
+
+
 .footer {
     background-color: var(--second-bg-color);
-    height: 10rem;
+    height: auto;
+    padding: 3rem 0 3rem 0;
 }
 
-.social:hover {
-    --webkit-transform: scale(1.1);
+.footer a {
+    /*color: var(--main-txt-color);*/
+    padding: 0 0.3rem 0 0.3rem;
 }
 
-.socal {
-    --webkit-transform: scale(0.8);
-    --webkit-transition-duration: 0.5s;
+#social-fb {
+
+    background: rgb(1, 50, 203);
+    background: linear-gradient(129deg, rgba(1, 50, 203, 1) 0%, rgba(80, 158, 247, 1) 100%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
+}
+
+#social-yt {
+    background: rgb(164, 0, 0);
+    background: linear-gradient(129deg, rgba(164, 0, 0, 1) 0%, rgba(254, 51, 51, 1) 100%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
+}
+
+#social-ig {
+    background: rgb(110, 1, 179);
+    background: linear-gradient(129deg, rgba(110, 1, 179, 1) 0%, rgba(240, 20, 16, 1) 32%, rgba(246, 254, 51, 1) 100%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
+}
+
+#social-em {
+    background: rgb(246, 79, 4);
+    background: linear-gradient(129deg, rgba(2226, 70, 0, 1) 25%, rgba(255, 139, 63, 1) 100%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
 }
 
 /*
     Multicoloured Hover Variations
 */
 
 #social-fb:hover {
-    color: #3b5998;
+
+    background: rgb(80, 158, 247);
+    background: linear-gradient(129deg, rgba(80, 158, 247, 1) 0%, rgba(1, 50, 203, 1) 70%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
 }
 
-#social-tw:hover {
-    color: #4099ff;
+#social-yt:hover {
+    background: rgb(254, 51, 51);
+    background: linear-gradient(129deg, rgba(254, 51, 51, 1) 0%, rgba(164, 0, 0, 1) 70%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
 }
 
-#social-gp:hover {
-    color: #d34836;
+#social-ig:hover {
+    background: rgb(246, 254, 51);
+    background: linear-gradient(129deg, rgba(246, 254, 51, 1) 8%, rgba(240, 20, 16, 1) 32%, rgba(110, 1, 179, 1) 90%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
 }
 
 #social-em:hover {
-    color: #f39c12;
+    background: rgb(255, 139, 63);
+    background: linear-gradient(129deg, rgba(255, 139, 63, 1) 5%, rgba(226, 70, 0, 1) 70%);
+    -webkit-text-fill-color: transparent;
+    -webkit-background-clip: text;
+}
+
+
+
+
+/* 
+:: TABLET STYLING 768px
+ */
+@media screen and (min-width: 600px) {
+
+    .description {
```

Comment:
When using relative units, media queries should also be relative. Use `em` where `1em = 16px` always (you can't make 1em = 10px). `em` is the only unit that works consistently across all browsers. [Know more about em in media queries](https://zellwk.com/blog/media-query-units/)

Reply:
Fixed it :)